Kudu.Core/SiteExtensions/SiteExtensionManager.cs

```diff
@@ -50,6 +50,8 @@ public class SiteExtensionManager : ISiteExtensionManager
         private readonly IContinuousJobsManager _continuousJobManager;
         private readonly ITriggeredJobsManager _triggeredJobManager;
 
+        private static readonly string _toBeDeletedDirectoryPath = Path.Combine(System.Environment.ExpandEnvironmentVariables(@"%HOME%\data\SiteExtensions"), "tobedeleted");
```

Comment:
we cannot, it is not on the same machine,  will result in access denied error.

i reference what kudusync did, it move data to "%HOME%\data\KuduSync